support/handling-tickets.md

```diff
@@ -0,0 +1,40 @@
+# How we handle support tickets
+Support tickets are logged in Lets Go Direct (LANDesk).
```

Comment:
Should't we split this out into each partner, until we get a shared helpdesk system?
E.g. for NCC & CCC use Landesk, for MKC use Hornbill

Reply:
We should probably just add a MKC specific line at the minute but based on our recent conversation, we'll keep it quite generic for now and a 'to be confirmed / changed'

Reply:
added [here](https://github.com/lgss/lgss-digital-tech-playbook/pull/42/commits/ee4f3090dbc6e6ec34800b2d2271123b53c9f531)